### DISCUSSION TITLE
Feature request: ability to react with emojis to lines of code when reviewing PR diff

### DISCUSSION THREAD

**Discussion by olivewong** [community/community#103055] [Pull Requests]
### Select Topic Area

General

### Body

When reviewing someone's PR, I would like to react emojis (like you would a text message) to individual lines of code. For example, if something is nice, ✨ or in other cases, a 🤔. (The current options of thumbs down/up/rocket on comments is too limited and intense, leading to combative tone, imo, there should be more options)

Leaving a comment of "✨" is not ideal because it would take space in discussion that may have other legitimate comments (and is also too much and too many clicks away). Emoji reactions could be displayed only on the diff view. They would serve the purpose of:
A) pointing out things to the author and other reviewers 
B) showing appreciation for work or implementation 
C) encouraging more lightweight interactions while reviewing code that may not require change or full comments

**Comment by Hexotical** [comment]
This would be amazing!

**Comment by jcolot** [comment]
Upvoted. It's a shame that such a large social network doesn't have this basic feature.
